src/main/java/com/rapid7/net/AsyncLogger.java

```diff
@@ -359,6 +360,11 @@ boolean checkValidUUID(String uuid) {
         return true;
     }
 
+    //  VisibleForTesting
```

Comment:
```suggestion
    @VisibleForTesting
```
?

Reply:
this is in guava, which isn't here so so I assume we don't want to import the world just for an annotation